packages/insured-bridge-relayer/src/Relayer.ts

```diff
@@ -727,66 +652,159 @@ export class Relayer {
+          message: "Slow Relay executed  🐌",
+          mrkdwn,
+        };
 
       case RelaySubmitType.SpeedUp:
         this.logger.debug({
-          at: "InsuredBridgeRelayer#Relayer",
+          at: "AcrossRelayer#Relayer",
           message: "Speeding up existing relayed deposit",
           realizedLpFeePct: realizedLpFeePct.toString(),
           relayableDeposit,
         });
-        if (pendingRelay === undefined)
+        if (pendingRelay === undefined) {
           // The `pendingRelay` should never be undefined if shouldRelay returns SpeedUp, but we have to catch the
           // undefined type that is returned by the L1 client method.
-          this.logger.error({ at: "InsuredBridgeRelayer#Relayer", message: "speedUpRelay: undefined relay" });
-        else await this.speedUpRelay(relayableDeposit.deposit, pendingRelay);
-        break;
+          this.logger.error({ at: "AcrossRelayer#Relayer", message: "speedUpRelay: undefined relay" });
+          return { transaction: null, message: "", mrkdwn: "" };
+        } else
+          return {
+            transaction: this._generateSpeedUpRelayTx(relayableDeposit.deposit, pendingRelay),
+            message: "Slow relay sped up 🏇",
+            mrkdwn,
+          };
+
       case RelaySubmitType.Instant:
         this.logger.debug({
-          at: "InsuredBridgeRelayer#Relayer",
+          at: "AcrossRelayer#Relayer",
           message: "Instant relaying deposit",
           realizedLpFeePct: realizedLpFeePct.toString(),
           relayableDeposit,
         });
-        await this.instantRelay(relayableDeposit.deposit, realizedLpFeePct);
-        break;
+        return {
+          transaction: this._generateInstantRelayTx(relayableDeposit.deposit, realizedLpFeePct),
+          message: "Relay instantly sent 🚀",
+          mrkdwn,
+        };
     }
   }
 
+  private _generateMarkdownForRelay(deposit: Deposit, realizedLpFeePct: BN) {
+    return (
+      "Relayed " +
+      this._generateMrkdwnDepositIdNetworkSizeFromTo(deposit) +
+      "slowRelayFeePct " +
+      createFormatFunction(2, 4, false, 18)(toBN(deposit.slowRelayFeePct).muln(100)) +
+      "%, instantRelayFeePct " +
+      createFormatFunction(2, 4, false, 18)(toBN(deposit.instantRelayFeePct).muln(100)) +
+      "%, realizedLpFeePct " +
+      createFormatFunction(2, 4, false, 18)(realizedLpFeePct.muln(100)) +
+      "%."
+    );
+  }
+
+  private _generateMarkdownForSettle(deposit: Deposit, relay: Relay) {
+    return (
+      "Settled " +
+      this._generateMrkdwnDepositIdNetworkSizeFromTo(deposit) +
+      this._generateMrkdwnForBonds(deposit, relay) +
+      this._generateMrkdwnForRelayerAddresses(deposit, relay) +
+      " ."
+    );
+  }
+
+  private _generateMrkdwnForDispute(deposit: Deposit, relay: Relay) {
+    return (
+      "Disputed " +
+      this._generateMrkdwnDepositIdNetworkSizeFromTo(deposit) +
+      "DepositHash " +
+      deposit.depositHash +
+      " relayAncillaryDataHash " +
```

Comment:
+1. This ancillary data can be matched against a SkinnyOO price request's ancillary data by parsing the price request's UTF8-decoded ancillary data.